packages/contracts/contracts/CollSurplusPool.sol

```diff
@@ -101,6 +116,7 @@ contract CollSurplusPool is ICollSurplusPool, ReentrancyGuard, AuthNoOwner {
         unchecked {
             totalSurplusCollShares = cachedTotalSurplusCollShares - claimableColl;
         }
+        emit SurplusCollSharesClaimed(_account, claimableColl);
```

Comment:
Do we need both events? They are basically the same

Reply:
removed this and sticking with the `CollSharesTransferred` event